geode-core/src/main/java/org/apache/geode/internal/cache/DiskStoreImpl.java

```diff
@@ -4564,4 +4569,13 @@ public StatisticsFactory getStatisticsFactory() {
     return this.cache.getDistributedSystem();
   }
 
+  public int getDiskUsagePercentage() {
+    long totalDiskSpace = 0;
+    for (DirectoryHolder dr : this.directories) {
+      totalDiskSpace += dr.getDiskDirectoryStats().getDiskSpace();
+    }
+    int usage = (int) (totalDiskSpace * 100 / this.totalDiskStoreSpace);
```

Comment:
Consider calculating and returning the percentage as a float instead of an int. I noticed there are a few other "Percentage" metrics in DiskStoreMXBean that are floats. To me, it makes more sense to do the calculation as a float because it's more accurate, but there might be valid reasons to use int instead.